Runtime/Core/ApiGatewayManagement/AnywhereGameServerAdapter.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I feel we've talked about this before. This is not going to support SDK4 right? If so, could we clean up LocalGameServerAdapter as well? Would be nice to include that file removal as part of this commit to keep all relevant changes bundled up.

Reply:
Removed these in another PR.